package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@google-cloud/logging-winston",
   "description": "Stackdriver Logging transport for Winston",
-  "version": "0.7.0",
+  "version": "0.6.0",
```

Comment:
Semver minor goes down?  Did we delete a feature?

Reply:
Luke did this upgrade and my file goes stale. Changed back.